jac-uk/apply
# unmerged 845

## What's included?
This PR is to un-merge ticket #845 - Separating QT code as it was merged by mistake. 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Log into Apply portal. Online Tests should be available within the apply portal itself, i.e. should not redirect away to the QT website.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work
🟠 Medium risk - this does change code that is shared with other areas
🔴 High risk - this includes a lot of changes to shared code

## Additional context
Include screen grabs, notes etc.

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
